cou/apps/app.py

```diff
@@ -245,24 +224,121 @@ def __str__(self) -> str:
             yaml.dump(summary, stream)
             return stream.getvalue()
 
+    def _populate_units(self) -> None:
+        """Populate application units."""
+        if not self.is_subordinate:
+            for name, unit in self.status.units.items():
+                compatible_os_version = (
+                    self._get_latest_compatible_os_version_by_unit_workload_version(unit)
```

Comment:
not-blocking: Is there chance to make name of this function shorter?

Reply:
Any suggestion?
maybe `_get_latest_os_version_by_workload_version`

Reply:
Still long I think, but better :+1:.